doctest.cabal

```diff
@@ -127,13 +127,15 @@ library
       Runner
       Runner.Example
       Sandbox
+      ThreadPool
```

Comment:
`package.yaml` is the canonical source of truth and `doctest.cabal` is a generated file.